compositor_render/src/renderer/transformations_registry.rs

```diff
@@ -0,0 +1,49 @@
+use std::sync::Arc;
+
+use compositor_common::renderer_spec::RendererId;
+
+use crate::{
+    registry::{RegisterError, RegistryType, RendererRegistry},
+    transformations::{
+        builtin::transformations::BuiltinTransformations, image_renderer::Image, shader::Shader,
+        web_renderer::WebRenderer,
+    },
+};
+
+use super::{WgpuCtx, WgpuError};
+
+pub(crate) struct TransformationsRegistry {
```

Comment:
We don't want to use transformation name anymore

Reply:
Do you have any suggestions on how could we name it?
I didn't want to use `entity` here, since in #85 entity refers to inputs and outputs as well.
I think collecting it in some structure cleans up logic and separates responsibilities. 
Naming it `RenderersRegistry` seems weird, when `renderer.rs` is sth different.

Reply:
How about `Registry`?

Reply:
The registry does not seem correct here because it's not registry itself, it holds othre registries., so maybe `renderers`, but to be honest I don't really see any benefit from having this struct.

I see that we might want to limit the number of fields in the top-level Renderer struct, so if that was your goal I would suggest removing register method and RegistryEntry type. They are only adding complexity.